internal/client/client.go

```diff
@@ -267,6 +267,22 @@ func (a *AuthApiClient) CreateClusterSpec(cmd *cobra.Command, regionInfoList []m
 	if err != nil {
 		return nil, err
 	}
+
+	//Some region for example us-west2 do not offer same instances as other regions, so we take the lower memory from all regions with the same number of core.
+	// if we have more than 1 region
+	if len(clusterRegionInfo) > 1 {
```

Comment:
@vmallepalli - do we have a special config for instance types in different environments for this region?

Reply:
in us-west2 , we default to n1-standard- types instead of n2-standard- types, but it’s the same in all environments.